tests/php/TemplatingTest.php

```diff
@@ -70,6 +70,60 @@ public function templateWithSingleMustacheVariable_ReplacesVariableWithGivenValu
 		assertThat( $result, is( equalTo( '<p>some value</p>' ) ) );
 	}
 
+	/**
+	 * @test
+	 */
+	public function templateWithVariableAndDiacritcsInValue_ReplacesVariableWithEncodedValue() {
+		$result = $this->createAndRender( '<p>{{value}}</p>', [ 'value' => 'inglés' ] );
+
+		assertThat( $result, is( equalTo( '<p>inglés</p>' ) ) );
+	}
+
+	/**
+	 * @test
+	 */
+	public function templateWithVariableAndValueInKorean_ReplacesVariableWithEncodedValue() {
+		$result = $this->createAndRender( '<p>{{value}}</p>', [ 'value' => '한국어' ] );
+
+		assertThat( $result, is( equalTo( '<p>한국어</p>' ) ) );
+	}
+
+	/**
+	 * @test
+	 */
+	public function templateWithVhtmlVariable_ReplacesVariableWithGivenValue() {
+		$result = $this->createAndRender(
+			'<div><div v-html="value"></div></div>',
+			[ 'value' => '<p>some value</p>' ]
+		);
+
+		assertThat( $result, is( equalTo( '<div><div><p>some value</p></div></div>' ) ) );
+	}
+
+	/**
+	 * @test
+	 */
+	public function templateWithVhtmlAndDiacritcsInValue_ReplacesVariableWithEncodedValue() {
+		$result = $this->createAndRender(
+			'<div><div v-html="value"></div></div>',
+			[ 'value' => '<p>inglés</p>' ]
+		);
+
+		assertThat( $result, is( equalTo( '<div><div><p>inglés</p></div></div>' ) ) );
+	}
+
+	/**
+	 * @test
+	 */
+	public function templateWithVhtmlAndValueInKorean_ReplacesVariableWithEncodedValue() {
+		$result = $this->createAndRender(
+			'<div><div v-html="value"></div></div>',
+			[ 'value' => '<p>한국어</p>' ]
+		);
+
+		assertThat( $result, is( equalTo( '<div><div><p>한국어</p></div></div>' ) ) );
```

Comment:
Not quite to the point of this bug, but we talked about is earlier... if `v-html="value"` is not part of the rendered mark-up there is no way for client-side JS code to find the respective node for data binding. So what is really the benefit of having both mustache and the v-html way of interpolating variables?